README.rst

```diff
@@ -23,6 +23,23 @@ The contents will be:
 - power colours
 - rms spectra
 
+Installation
+-------------
```

Comment:
Yes, `develop`  should be in how to contribute section.

What I think is overwork for installing requirements first and then package itself.
Maybe add it to setup.py:
```
import os
from setuptools import setup

with open('requirements.txt') as f:
    required = f.read().splitlines()

setup(...
install_requires=required,
...)
```